starter-repos/app-template/config/envs/prod-central/varReference.yaml

```diff
@@ -0,0 +1,4 @@
+varReference:
```

Comment:
What's this for?

Reply:
This is to replace $(NAMESPACE) with the correct namespace in the ConfigMap. By default, this replacement can't happen in a ConfigMap. 

The key, value `REDISHOST=redis-master.$(NAMESPACE).svc.cluster.local` in the configmap is to make sure the application can talk to the redis server without knowing its cluster IP. The `$(NAMESPACE)` is used to make sure when users update `namespace: xxx` field in kustomization.yaml(or when `appctl env add` overwrites this namespace), they don't need to update other fields.